src/category.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+function Category() {
+  const category = useSelector((state) => state.category);
+  const dispatch = useDispatch();
+  const checkStatus = () => {
+    dispatch({ type: 'CHECK' });
+  };
+  return (
+    <div>
```

Comment:
### Optional
- [ ] For a better user experience it would be better if we import the `Headers` so that navigating the page will be so much easier. Here is how it should look for a quick fix
```
<>
      <Headers />
      <div>
        {category}
        <button type="button" onClick={checkStatus} id="check-status">Check Status</button>
      </div>
    </>

```
Remember to import the `Headers`